app/views/layouts/application.html.erb

```diff
@@ -1,15 +1,22 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <title><%= full_title(yield(:title)) %></title>
     <title>SampleApp</title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```

Comment:
Use " intead of ' in this whole file, only one space between stylesheet_link_tag , application